### DISCUSSION TITLE
Server Console - Up Arrow for last command

### DISCUSSION THREAD

**Discussion by Saamstep** [pufferpanel/pufferpanel#1024] [Ideas]
Similar to how a command line or terminal window works, when you press the up arrow button the input box will be populated with the command last ran. This is something I use quite frequently and would be a nice quality of life feature for PufferPanel.

**Comment by nepcore** [comment]
as in within one session or as in past a page reload as well? because the latter is rather hard especially when considering multiple users

**Reply by Saamstep** [reply]
Within one session is suitable in my opinion

**Reply by nepcore** [reply]
as a history can take up a good bit of additional memory if it grows too long i'd still want to limit it's length, would you ever see any need for more than 100 history entries? imo that should cover everything while still making sure that lower end clients don't crash due to lack of memory
